src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,532 @@
+            elif relation == 'amidation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein amidation',
+                        namespace='GO',
+                        identifier='0018032',
+                    ),
+                )
+            # protein deamination (negative effect, increases the deaminated target protein abundance)
+            elif relation == 'deamination reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein deamination',
+                        namespace='GO',
+                        identifier='0018032',
+                    ),
+                )
+            # ampylation reaction
+            elif relation == 'ampylation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein adenylylation',
+                        namespace='GO',
+                        identifier='0018117',
+                    ),
+                )
+            # myristoylation reaction
+            elif relation == 'myristoylation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein myristoylation',
+                        namespace='GO',
+                        identifier='0018377',
+                    ),
+                )
+            # lipid addition
+            elif relation == 'lipid addition':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='lipid binding',
+                        namespace='GO',
+                        identifier='0008289',
+                    ),
+                )
+                graph.add_increases(
+                    source,
+                    target_mod,
+                    citation=pubmed_id,
+                    evidence=EVIDENCE,
+                    subject_modifier=pybel.dsl.activity(),
+                )
+                continue
+            # aminoacylation reaction (tRNA-ligase activity)
+            elif relation == 'aminoacylation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='tRNA aminoacylation',
+                        namespace='GO',
+                        identifier='0043039',
+                    ),
+                )
+
+                graph.add_increases(
+                    source,
+                    target_mod,
+                    citation=pubmed_id,
+                    evidence=EVIDENCE,
+                    object_modifier=pybel.dsl.activity(),
+                )
+                continue
+            # dna strand elongation
+            elif relation == 'dna strand elongation':
```

Comment:
doesn't seem like it makes sense that the target is even a protein at all

Reply:
This is why I used 'pybel.dsl.Gene'. Is this not applicable here?

Reply:
yeah but DNA strand elongation is a process, not a protein or a protein modification. It's not really obvious what we should do here.

BUT WHAT IF

`pybel.dsl.GeneModification()` nailed ittt

Reply:
Nice, I adapted it